documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+
+# Create the writer, which we'll be using to save dataset-related files.
+output_dir = "."
+dataset_name = "processed_dataset"
+writer = ColorizerDatasetWriter(output_dir, dataset_name)
+```
+
+#### 3. Write the core data
+
+```python
+# Turn core data columns into a numpy array, to be saved by the writer.
+tracks = data[TRACK_ID_COLUMN].to_numpy()
+times = data[TIMES_COLUMN].to_numpy()
+centroids_x = data[CENTROIDS_X_COLUMN].to_numpy()
+centroids_y = data[CENTROIDS_Y_COLUMN].to_numpy()
+
+writer.write_data(
+    tracks=tracks,
+    times=times,
+    centroids_x=centroids_x,
+    centroids_y=centroids_y,
+)
+```
+
+#### 4. Write the features
+
+Features can be one of three types:
+
+1. **Continuous** features are used for floating-point numbers.
+2. **Discrete** features are used for integers.
+3. **Categorical** features are used for string-based labels. Note that there can only be a max of 12 categories for a categorical feature.
+
+The `FeatureInfo` class is used to provide metadata about each feature, such as its label, key, type, and units if applicable.
+
+```python
+areas = data[AREA_COLUMN].to_numpy()
+locations = data[LOCATION_COLUMN].to_numpy()
+radii = data[RADIUS_COLUMN].to_numpy()
+
+# Additional metadata can be provided for each feature, which will be shown
+# when interacting with it in the viewer.
+area_info = FeatureInfo(
+    label="Area",
+    key="area",
+    type=FeatureType.CONTINUOUS,
+    unit="px²",
+)
+radius_info = FeatureInfo(
+    label="Radius",
+    key="radius",
+    # Discrete features are used for integers.
+    type=FeatureType.DISCRETE,
+    unit="px",
+)
+location_info = FeatureInfo(
+    label="Location",
+    key="location",
+    # Categorical features are used for string-based labels.
+    type=FeatureType.CATEGORICAL,
+    # Categories can be auto-detected from the data, or provided manually
+    # if you want to preserve a specific order for the labels.
+    categories=["top", "middle", "bottom"],
+)
+writer.write_feature(areas, area_info)
+writer.write_feature(radii, radius_info)
+writer.write_feature(locations, location_info)
+```
+
+#### 5. Write the images
+
+The `ColorizerDatasetWriter` writes images as PNGs with encoded object IDs. You can see more about what this looks like and how it works in our [data format documentation](./DATA_FORMAT.md#5-frames). As previously noted, all object IDs must be unique too, so this next section will perform three tasks:
+
+1. Load in the image data from the segmentation images
+2. Remap the object IDs to be unique across all timepoints
+3. Write the remapped images using `ColorizerDatasetWriter.write_image()`
+
+```python
+# Group data by the timestamp
+data_grouped_by_time = data.groupby(TIMES_COLUMN)
+frame_paths = []
+
+for frame_num, frame_data in data_grouped_by_time:
+    # Get the path to the image and load it.
+    frame_path = frame_data.iloc[0][SEGMENTED_IMAGE_COLUMN]
+    segmentation_image = BioImage("raw_dataset/" + frame_path).get_image_data(
```

Comment:
Nit: maybe this should use pathlib and save "raw_dataset" as a constant so this code looks like:
```suggestion
    segmentation_image = BioImage(source_dataset_dir / frame_path).get_image_data(
```

Reply:
Changed, thanks!